tests/NHSD.BuyingCatalogue.Organisations.Api.IntegrationTests/Features/Organisations DeleteRelatedOrganisation.feature

```diff
@@ -27,6 +27,21 @@ Scenario: An authority user can delete an existing Organisation relationship
     And a response with an empty body is returned
 
 @5151
+Scenario: An authority user can delete an existing Organisation relationship that will not effect another relationship of the same type
+    Given a user is logged in
+        | Username             | Password        | Scope        |
+        | PostmanPat@email.com | An0therPa$$w0rd | Organisation |
+        And Organisation PrimaryOrganisation has a Parent Relationship to Organisation RelatedOrganisation
+        And Organisation UnrelatedOrganisation has a Parent Relationship to Organisation RelatedOrganisation
```

Comment:
We don't normally indent the `And` set-up steps. Right now I can't decide if actually like it or not! However, let's keep the formatting consistent for now.